backend/src/v2/compiler/argocompiler/dag.go

```diff
@@ -555,6 +556,7 @@ func (c *workflowCompiler) addDAGDriverTemplate() string {
 		"--http_proxy", proxy.GetConfig().GetHttpProxy(),
 		"--https_proxy", proxy.GetConfig().GetHttpsProxy(),
 		"--no_proxy", proxy.GetConfig().GetNoProxy(),
+		"--cache_enabled", strconv.FormatBool(*c.cacheEnabled),
```

Comment:
Optional: It'd be nice to only pass this argument if `*c.cacheEnabled` is false so that there is only an additional argument in the compiled Argo Workflow in the non-default case.

Reply:
Done.